playwright/tests/article-inline-slots.spec.ts

```diff
@@ -8,46 +8,90 @@ const pages = articles.filter(({ name }) => name === 'inlineSlots');
 
 test.describe('Slots and iframes load on article pages', () => {
 	pages.forEach(
-		({
-			path,
-			expectedMinInlineSlotsOnDesktop,
-			expectedMinInlineSlotsOnMobile,
-		}) => {
-			breakpoints.forEach(({ breakpoint, width, height }, index) => {
+		(
+			{
+				path,
+				expectedMinInlineSlotsOnDesktop,
+				expectedMinInlineSlotsOnMobile,
+				expectedSlotIndicesOnMobile,
+				expectedSlotIndicesOnDesktop,
+			},
+			index,
+		) => {
+			breakpoints.forEach(({ breakpoint, width, height }) => {
 				const expectedMinSlotsOnPage =
 					(breakpoint === 'mobile'
 						? expectedMinInlineSlotsOnMobile
 						: expectedMinInlineSlotsOnDesktop) ?? 999;
 
-				test(`Test article ${index} has at least ${expectedMinSlotsOnPage} inline total slots at breakpoint ${breakpoint}`, async ({
-					page,
-				}) => {
-					await page.setViewportSize({
-						width,
-						height,
-					});
+				const expectedSlotIndices =
+					breakpoint === 'mobile'
+						? expectedSlotIndicesOnMobile
+						: expectedSlotIndicesOnDesktop;
+
+				if (!expectedSlotIndices) {
```

Comment:
Is this code block still needed? It looks like indices are defined for all pages we're testing.

Is `positions` a more descriptive word than `indices` here? I'm not sure. Will leave that one up to you.

Reply:
THis is just making typescript happy as far as I can remember, positions is a better term, cheers